pkg/handler/nfts.go

```diff
@@ -139,7 +139,22 @@ func (h *Handler) CreateNFTSalesTracker(c *gin.Context) {
 		return
 	}
 
-	err := h.entities.CreateNFTSalesTracker(req.ContractAddress, req.Platform, req.GuildID)
+	checksum, err := util.ConvertToChecksumAddr(req.ContractAddress)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+
+	err = h.entities.UpsertSalesTrackerConfig(request.UpsertSalesTrackerConfigRequest{
+		GuildID:   req.GuildID,
+		ChannelID: req.ChannelID,
+	})
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+
+	err = h.entities.CreateNFTSalesTracker(checksum, req.Platform, req.GuildID)
```

Comment:
in entities layer, add log for error
can ref to `e.log.Errorf(err, "Error in fail .... %v", req.Address)`